prow/config.yaml

```diff
@@ -606,6 +606,55 @@ presubmits:
         - name: cache-ssd
           hostPath:
             path: /mnt/disks/ssd0
+
+  - name: pull-kubernetes-bazel-build-canary
+    agent: kubernetes
+    context: pull-kubernetes-bazel-build-canary
+    always_run: false
+    rerun_command: "/test pull-kubernetes-bazel-build-canary"
+    trigger: "(?m)^/test pull-kubernetes-bazel-build-canary,?(\\s+|$)"
+    skip_branches:
```

Comment:
nit: also `branches: master` to unify with the test job?

Reply:
thanks, fixed!